scripts/lokalise/lokalise_client.rb

```diff
@@ -31,6 +31,9 @@ def create_key(key_object)
                 {
                     "key_name": {
                         "android": key_object[:key_name],
+                        "ios": key_object[:key_name],
+                        "web": key_object[:key_name],
+                        "other": key_object[:key_name],
```

Comment:
Missed this in the initial Lokalise work: When we create a new key, Lokalise seems to expect values for _all_ platforms.